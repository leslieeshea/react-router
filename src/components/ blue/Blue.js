import React from 'react';

export default function Blue() {
  const style = {
    backgroundColor: 'blue',
    width: '50vh',
    height: '50vh'
  };

  return <div style={style}></div>;
}
