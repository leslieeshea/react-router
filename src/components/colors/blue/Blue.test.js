import React from 'react';
import { shallow } from 'enzyme';
import Blue from './Blue';

describe('Blue component', () => {
  it('renders a blue splotch', () => {
    const wrapper = shallow(<Blue />);
    expect(wrapper).toMatchSnapshot();
  });
});
