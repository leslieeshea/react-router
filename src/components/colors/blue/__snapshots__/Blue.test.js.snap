// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blue component renders a blue splotch 1`] = `
<div
  style={
    Object {
      "backgroundColor": "blue",
      "height": "50vh",
      "width": "50vh",
    }
  }
/>
`;
